src/components/Content/Content.js

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { Eye, MessageCircle, Heart, Image, Paperclip, Video  } from 'react-feather';
+import './Content.css';
+
+function Content() {
+  return (
+  <main>
+    <section className="cards">
+      <div className="card">
+        <div className="card_image-container">
+          <img src="https://images.unsplash.com/photo-1520190282873-afe1285c9a2a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1200&q=80" alt="cabin ontop of mountain"
+          />
+          <div class="overlay">
+            <span className="text">Mountain retreats</span>
+          <button type="button"><a href="save-button" class="save-button">Save</a></button>
```

Comment:
This is not semantic. An `a` tag and a `button` are two different things. `a` tags link to things, `buttons` represent actions that are taken by the user. It makes sense that this is a button, not an `a`. You cannot place an `a` in a button or vice-versa.

Reply:
Thanks-that helped a lot.